### PR TITLE
ACTS vectorized LA backend project proposal

### PR DESCRIPTION
**Pull request by niermann999**
ACTS project proposal for vectorized linear algebra backend (co-mentors @HadrienG2 and @paulgessinger)

### CODE REVIEW

File: _gsocproposals/2022/proposal_Acts-vectorized-LA-backend.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: Vectorized linear algebra implementation for Acts
+layout: gsoc_proposal
+project: Acts
+year: 2022
+organization: CERN
```
Comment (hegner) [inline _gsocproposals/2022/proposal_Acts-vectorized-LA-backend.md:6]: maybe you want to add IJCLab ?

File: _gsocproposals/2022/proposal_Acts-vectorized-LA-backend.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+* Adapt the implementation to the linear algebra operations needed in the ACTS numerical integration for particle propagation.
+* Validate the approach and its performance and possibly optimize the performance further.
+* Given enough time, find a way to make the implementation available to ACTS as a compute backend and run it in ACTS example code.
+
+## Expected results
+Working prototype implementation that contains all necessary operations and performance statement
+
+## Requirements
+C++, experience with code optimization/performance tools, such as perf or vtune, and vectorization would be an asset
+
```
Comment (hegner) [inline _gsocproposals/2022/proposal_Acts-vectorized-LA-backend.md:33]: Could you add a piece about an "Evaluation Task", which is part of a full project proposal?

**Review by hegner — CHANGES_REQUESTED** [review]

File: _gsocorgs/2022/ijclab.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+title: "Laboratoire de Physique des 2 infinis Irène Joliot-Curie (IJCLab)"
+author: "Julien Peloton"
```
Comment (aflp91) [inline _gsocorgs/2022/ijclab.md:3]: It is strange to see here Julien's name as he is not involved in GSoC this year? Maybe, if a name is needed, Hadrien's name is more appropriate?

File: _gsocproposals/2022/proposal_Acts-vectorized-LA-backend.md (hunk 2 above)
Comment (niermann999) [reply _gsocproposals/2022/proposal_Acts-vectorized-LA-backend.md:38]: Done

File: _gsocproposals/2022/proposal_Acts-vectorized-LA-backend.md (hunk 1 above)
Comment (niermann999) [reply _gsocproposals/2022/proposal_Acts-vectorized-LA-backend.md]: done

File: _gsocorgs/2022/ijclab.md (hunk 3 above)
Comment (niermann999) [reply _gsocorgs/2022/ijclab.md]: I added Hadrien now

**Review by hegner — APPROVED** [review]

**Pull request merged**